src/layouts/ProviderLayout.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <q-layout view="lHh Lpr lFf">
```

Comment:
Can I ask why the view has those weird characters ("lHh Lpr lFf")? Does it mean something?

Reply:
`q-layout` component use that to define the layout, that doesn't likes me, I've thought to use another way to define the layout of the APP when I need to add styles to the layout

Reply:
You can use the layout builder https://quasar.dev/layout-builder

Or check this documentation https://quasar.dev/layout/layout

if you want to use "lHh Lpr lFf" and understand what it meant

Reply:
Please post in the code some documentation URL.

I agree is weird.